crates/key-server/src/tests/e2e.rs

```diff
@@ -386,6 +392,160 @@ async fn test_e2e_imported_key() {
         .is_err_and(|e| e == UnsupportedPackageId));
 }
 
+#[traced_test]
+#[tokio::test]
+async fn test_e2e_mpc() {
```

Comment:
this test uses the real result from dkg-cli run (see readme for test data artifacts). and run all the move txn (for init, register, propose, approve, finalize). skim to review overall steps